src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs

```diff
@@ -37,6 +38,16 @@ public ConsoleParameters(IFileHelper fileHelper)
             this.fileHelper = fileHelper;
         }
 
+#if NET451
+
+        /// <summary>
+        /// TODO: Remove the #if when project is targeted to netstandard2.0
+        /// Environment variables to be set for the process
+        /// </summary>
+        public StringDictionary EnvironmentVariables { get; set; } = new StringDictionary();
```

Comment:
Should this just be IDictionary<string, string> instead? Looks like StringDictionary is pre generics from [here](https://stackoverflow.com/questions/627716/stringdictionary-vs-dictionarystring-string).

Reply:
agree

---
In reply to: [287176793](https://github.com/microsoft/vstest/pull/2023#discussion_r287176793) [](ancestors = 287176793)